### DISCUSSION TITLE
Shop packages QoL

### DISCUSSION THREAD

**Discussion by JXYZP-G** [gameserverapp/Platform#345] [Suggestions]
Hey i have been using this shop system for a while and i feel like there should be more options how frequently a person can purchase a item.

![image](https://user-images.githubusercontent.com/43095469/218332239-a54e73a3-14fd-4ef6-b75b-908a857098ef.png)

As example:
- Limit it to a fixed amount and after that you have to pay tokens(custom amount)
- A separate option how how many days before refill 
- Or you can claim 1 shop item for a specific price then locks up till u purchase a support tier.
- Require a ingame level before purchase
- A Custom link when you press "Get Tokens" in the ingame menu.
- Possibility to trade tokens between players.
- Reset Amount of free packes(for example at wipe resets everyone back at 3 if default free packages where 3)

If anyone else has more please reply on this post. Thanks!

**Comment by mvaessen** [comment]
Thanks for posting a nicely detailed suggestion! I like that 👍
